src/ballet/base58/BUILD

```diff
@@ -0,0 +1,33 @@
+load("//bazel:fd_build_system.bzl", "fd_cc_library", "fd_cc_test")
+
+package(default_visibility = ["//src/ballet:__subpackages__"])
+
+fd_cc_library(
+    name = "base58",
+    srcs = [
+        "fd_base58.c",
+        "fd_base58.inc",
+        "fd_base58_avx.inc",
+        "fd_base58_avx.h",
```

Comment:
Not sure if this is kosher. I don't want the avx header exposed as an actual header. It's designed for internal use only.